frontend/src/components/layout/navigation/whatsnew/WhatsNewMenu.module.scss

```diff
@@ -1,21 +1,24 @@
-@import "../../../styles/tokens.scss";
+@import "../../../../styles/tokens.scss";
 
-.trigger {
+button.trigger {
```

Comment:
:thinking: Hmm, using CSS modules I would expect something like this not to be necessary? Do you happen to remember what went wrong without the `button`?

Reply:
Specificity related - I believe its because there is styling coming from `layout` - without it, it looks like the below. 

<img width="808" alt="image" src="https://user-images.githubusercontent.com/3924990/170298320-3d31e289-e997-463a-bb85-f0ce558867f7.png">